### ISSUE TITLE
Reword Blocking request DEBUG Message

### ISSUE THREAD

**Issue by jwulf** [zeebe-io/zeebe#3890]
This log message is frequently misinterpreted by users as an error:

```
DEBUG io.zeebe.gateway - Jobs of type payment-service not available. Blocking request BrokerActivateJobsRequest{requestDto={"type":"payment-service","worker":"default","timeout":300000,"maxJobsToActivate":32,"jobKeys":[],"jobs":[],"variables":[],"truncated":false}}
```
This leads them down the wrong trail when they are learning to use the system and it is not working as they expect.

**Describe the solution you'd like**

Change this message to say:

```
DEBUG io.zeebe.gateway - A worker asked for jobs of type "payment-service". There are no jobs of this type to work on right now. 
```

I'm not sure what `Blocking request BrokerActivateJobsRequest` communicates, but it could be changed to something that makes sense. At the moment, a user is thinking: 

> Something is wrong. I don't know what it is. Maybe this? This doesn't look good. It says something is blocking my jobs from running.

Does it mean that this request is now blocking something, or is that the request that blocked something? Or that the broker blocked this request?

Even I don't know.

We may need this for debugging purposes, and we should also make it clear in the language domain of new users what is being communicated.

**Comment by grexe** [comment]
exactly fell into that trap, please use a less dramatic wording both for worker and for broker...

**Comment by Zelldon** [comment]
I think we could think of just dropping this message. For me it is mostly just noise.

**Comment by korthout** [comment]
When looking into the issue, we noticed the log level was already lowered to TRACE. We think the message does provide valuable information, but not all the time. So we chose to keep a message at that level.

The new message should provide more structured information, and clarify what is meant better. The new message describes the situation at that moment (no jobs found for type after a request by a specific worker), and describes the action taken by the system in response (request is kept open until new job or timeout).
